### PR TITLE
Add city Potsdam

### PR DESCRIPTION
**Pull request by hfs**

### CODE REVIEW

File: cities/potsdam.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        {
+            "geometry": {
+                "coordinates": [
+                    13.052036762237547,
+                    52.400651475660936
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "location": "Lindenstra\u00dfe 53/56",
```
Comment (johnjohndoe) [inline cities/potsdam.json]: Are these characters on purpose?

Comment (hfs) [inline cities/potsdam.json]: Ugh, no. I’ll reformat it to use plain UTF-8 (later).

The escape sequences are a result of using `python -mjson.tool` to pretty-print the JSON, by the way.

**Comment by hfs** [comment]
I’ve updated the branch based on your comments. Please give it another shot.

**Comment by johnjohndoe** [comment]
@hfs Thank you for adding markets in Potsdam. Please let the public know: a tweet with screenshot and web address would be great. Further, if you could convince the city of Potsdam to link the website - that would be great. See #38.

**Comment by hfs** [comment]
Tweet: https://twitter.com/oklabpdm/status/809331056847572992

**Pull request merged**